packs/extinction-curse-bestiary/flea-swarm.json

```diff
@@ -48,7 +48,7 @@
                 },
                 "category": "offensive",
                 "description": {
-                    "value": "<p><strong>Requirements</strong> The flea swarm's last action was Swarming Bites, and it dealt damage to at least one enemy</p><hr /><p><strong>Effect</strong> The flea swarm drinks the creatures' blood. Each victim becomes @UUID[Compendium.pf2e.conditionitems.Item.Drained]{Drained 1} and the swarm regains @Damage[3[healing]] HP for each victim drained.</p>"
+                    "value": "<p><strong>Requirements</strong> The flea swarm's last action was Swarming Bites, and it dealt damage to at least one enemy</p>\n<hr />\n<p><strong>Effect</strong> The flea swarm drinks the creatures' blood. Each victim becomes @UUID[Compendium.pf2e.conditionitems.Item.Drained]{Drained 1} and the swarm regains [[/r 3[healing]]]{3 HP} for each victim drained.</p>"
```

Comment:
Is there a reason to use /r here specifically?

Reply:
No, that shouldn't have changed...
Weird. I'll revert it back.